client/src/components/NavBar/NavBar.tsx

```diff
@@ -0,0 +1,90 @@
+import React, { useState } from 'react';
+import { User } from '../../interface/User';
+import useStyles from './useStyles';
+import AppBar from '@material-ui/core/AppBar';
+import Toolbar from '@material-ui/core/Toolbar';
+import Typography from '@material-ui/core/Typography';
+import IconButton from '@material-ui/core/IconButton';
+import MenuIcon from '@material-ui/icons/Menu';
+import AddOutlinedIcon from '@material-ui/icons/AddOutlined';
+import Container from '@material-ui/core/Container';
+import Grid from '@material-ui/core/Grid';
+import Button from '@material-ui/core/Button';
+import AvatarDisplay from '../../components/AvatarDisplay/AvatarDisplay';
+import CalendarTodayOutlinedIcon from '@material-ui/icons/CalendarTodayOutlined';
+import DashboardOutlinedIcon from '@material-ui/icons/DashboardOutlined';
+import logo from '../../Images/logo.png';
+
+interface Props {
+  loggedInUser: User;
+  handleDrawerToggle?: () => void;
+}
+
+const NavBar = ({ loggedInUser }: Props): JSX.Element => {
+  const classes = useStyles();
+  const [auth, setAuth] = useState(true);
+  const [anchorEl, setAnchorEl] = useState<null | HTMLElement>(null);
+  const open = Boolean(anchorEl);
+
+  const handleChange = (event: React.ChangeEvent<HTMLInputElement>) => {
+    setAuth(event.target.checked);
+  };
+
+  const handleMenu = (event: React.MouseEvent<HTMLElement>) => {
+    setAnchorEl(event.currentTarget);
+  };
+
+  const handleClose = () => {
+    setAnchorEl(null);
+  };
+
+  return (
+    <Container className={classes.root}>
+      <Grid container spacing={4} direction="row" alignItems="center" justify="center" className={classes.statusBar}>
+        <Grid item xs={4}>
+          <Grid container spacing={1} direction="row" alignItems="center" justify="flex-start">
+            <img alt="Kanban logo" src={logo} />
+          </Grid>
+        </Grid>
+        <Grid item xs={4}>
+          <Grid container spacing={3} alignItems="center" justify="space-evenly">
+            <Grid item>
+              <Button color="primary" size="large" startIcon={<CalendarTodayOutlinedIcon />}>
+                Dashboard
+              </Button>
+            </Grid>
+            <Grid item>
+              <Button color="primary" size="large" startIcon={<DashboardOutlinedIcon />}>
+                Calendar
+              </Button>
+            </Grid>
+          </Grid>
+        </Grid>
+        <Grid item xs={4}>
+          <Grid container spacing={3} alignItems="center" justify="space-evenly">
+            <Grid item>
+              <Button color="primary" variant="contained" size="large" startIcon={<AddOutlinedIcon />}>
```

Comment:
In the mock-up there is no shadow on the button. The button can get a `disableElevation` property and that will follow the mock-up a bit more.

Reply:
Updated in below commit. Good eye!